tests/CustomTaxonomy/TaxonomyCliTest.php

```diff
@@ -31,56 +35,142 @@
 	deleteCliOutput();
 });
 
+test('Custom taxonomy CLI command will correctly copy the Custom taxonomy class with defaults', function () {
+	$tax = $this->tax;
+	$tax([], $tax->getDevelopArgs([]));
+
+	// Check the output dir if the generated method is correctly generated.
+	$generatedCPT = \file_get_contents(\dirname(__FILE__, 3) . '/cliOutput/src/CustomTaxonomy/LocationTaxonomy.php');
+
+	$this->assertStringContainsString('class LocationTaxonomy extends AbstractTaxonomy', $generatedCPT);
+	$this->assertStringContainsString('location', $generatedCPT);
+	$this->assertStringContainsString('post', $generatedCPT);
```

Comment:
```suggestion
	expect(generatedCPT)
		->toContain('class LocationTaxonomy extends AbstractTaxonomy')
		->toContain('location')
		->toContain('post');	
```

Expectation API is prettier 😄

Reply:
👍